wg-api/spec-documents/scroll-view.md

```diff
@@ -0,0 +1,256 @@
+
+Process: **Main**
+
+### `new ContainerView()`
+
+Creates the new container view.
+
+### Instance Methods
+
+#### `containerView.addChild(view)`
+
+#### `containerView.removeChild(view)`
+
+#### `containerView.getChildren()`
+
+Returns `BaseView[]` - an array of all BaseViews that have been attached
+with `addChild`.
+
+
+## ScrollView
+
+The `ScrollView` can show an arbitrary content view inside it. It is used to make
+any View scrollable. When the content is larger than the `ScrollView`,
+scrollbars will be optionally showed. When the content view is smaller
+then the `ScrollView`, the content view will be resized to the size of the
+`ScrollView`.
+It extends `BaseView`.
+
+Process: **Main**
+
+### `new ScrollView()`
+
+Creates the new scroll view.
+
+### Instance Methods
+
+#### `scrollView.setContentView(contents)`
+
+Set the contents. The contents is the view that needs to scroll.
+
+#### `scrollView.getContentView()`
+
+Returns `BaseView` - The contents of the `scrollView`.
+
+#### `scrollView.setContentSize(size)`
+
+Set the size of the contents.
+
+#### `scrollView.getContentSize()`
+
+Returns the `size` of the contents.
+
+#### `scrollView.setHorizontalScrollBarMode(mode)`
+
+* `mode` string - Can be one of the following values: `disabled`, `hidden-but-enabled`, `enabled`. Default is `enabled`.
+
+Controls how the horizontal scroll bar appears and functions.
+* `disable` - The scrollbar is hidden.
+* `hidden-but-enabled` - The scrollbar is hidden whether or not the contents are larger than the viewport, but the pane will respond to scroll events. This option is not available on MacOS.
+*`enabled` - The scrollbar will be visible if the contents are larger than the viewport.
+
+#### `scrollView.setVerticalScrollBarMode(mode)`
+
+Controls how the vertical scroll bar appears and functions.
+
+#### `getHorizontalScrollBarMode()`
+
+Returns `string` - horizontal scrollbar mode.
+
+#### `scrollView.getVerticalScrollBarMode()`
+
+Returns `string` - vertical scrollbar mode.
+
+#### `scrollView.setHorizontalScrollElasticity(elasticity)` _macOS_
+
+* `elasticity` string - Can be one of the following values: `automatic`, `none`, `allowed`. Default is `automatic`.
+
+The scroll view’s horizontal scrolling elasticity mode.
+A scroll view can scroll its contents past its bounds to achieve an elastic effect. 
+When set to `automatic`, scrolling the horizontal axis beyond its document
+bounds only occurs if the document width is greater than the view width,
+or the vertical scroller is hidden and the horizontal scroller is visible.
+* `automatic` - Automatically determine whether to allow elasticity on this axis.
+* `none` - Disallow scrolling beyond document bounds on this axis.
+*`allowed` - Allow content to be scrolled past its bounds on this axis in an elastic fashion.
+
+#### `scrollView.setVerticalScrollElasticity(elasticity)` _macOS_
```

Comment:
docs: need to copy the `Can be ...` section here too.

Reply:
fixed

Reply:
fixed in 372905c240d9fc6953d2e7aefca0b1d6fcd2b795